code/datums/diseases/advance/symptoms/sensory.dm

```diff
@@ -62,4 +62,4 @@ Bonus
 					M.reagents.add_reagent("oculine", 20)
 			else
 				if(prob(SYMPTOM_ACTIVATION_PROB * 5))
-					to_chat(M, "<span class='notice'>[pick("Your eyes feel great.","You feel like your eyes can focus more clearly.", "You don't feel the need to blink.","Your ears feel great.","Your healing feels more acute.")]</span>")
+					to_chat(M, "<span class='notice'>[pick("Your eyes feel great.","You feel like your eyes can focus more clearly.", "You don't feel the need to blink.","Your eyes feel great.","Your hearing feels more acute.")]</span>")
```

Comment:
both `Your ears feel great` and `Your eyes feel great` fit context wise since oculine fixes both.
Can add another entry if you wish or leave it like this. Gonna leave it up to you

Reply:
"your eyes feels great" makes more sense in my opinion, though i see no problem in having both.